azure/templates/virtualMachine.json

```diff
@@ -34,6 +34,7 @@
         "osProfile": {
           "computername": "[concat(parameters('vmSetting').vmName,'-',copyIndex())]",
           "adminUsername": "[parameters('vmSetting').adminUsername]",
+          "customData": "[base64(concat('MARKLOGIC_LICENSE_KEY=',parameters('vmSetting').licenseKey,'\nMARKLOGIC_LICENSEE=',parameters('vmSetting').licensee,'\nMARKLOGIC_HOSTNAME=',reference(resourceId('Microsoft.Network/publicIpAddresses/',concat(parameters('vmSetting').nodePublicIpNameV4,copyIndex())), '2017-11-01').dnsSettings.fqdn,'\nMARKLOGIC_NODE_NAME=',reference(resourceId('Microsoft.Network/publicIpAddresses/',concat(parameters('vmSetting').nodePublicIpNameV4,copyIndex())), '2017-11-01').dnsSettings.fqdn,'\nMARKLOGIC_ADMIN_USERNAME=',parameters('vmSetting').mlAdminUsername,'\nMARKLOGIC_ADMIN_PASSWORD=',parameters('vmSetting').mlAdminPassword,'\nMARKLOGIC_AZ_DISK=',parameters('vmSetting').azDisk))]",
```

Comment:
We don't need both `MARKLOGIC_HOSTNAME` and `MARKLOGIC_NODE_NAME`. To be consistent with AWS, we will just use `MARKLOGIC_NODE_NAME`.